### PR TITLE
More unit fixes

### PR DESCRIPTION
**Pull request by amilcarlucas**
Fixes and adds units attributes
Adds display="bitmask" attribute in some more bitmask fields

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -3348,7 +3348,7 @@
       <field type="uint16_t" name="command" enum="MAV_CMD">Command ID, as defined by MAV_CMD enum.</field>
       <field type="uint8_t" name="result" enum="MAV_RESULT">See MAV_RESULT enum</field>
       <extensions/>
-      <field type="uint8_t" name="progress">WIP: Also used as result_param1, it can be set with a enum containing the errors reasons of why the command was denied or the progress percentage or 255 if unknown the progress when result is MAV_RESULT_IN_PROGRESS.</field>
+      <field type="uint8_t" name="progress" units="%">WIP: Also used as result_param1, it can be set with a enum containing the errors reasons of why the command was denied or the progress percentage or 255 if unknown the progress when result is MAV_RESULT_IN_PROGRESS.</field>
```
Comment (WickedShell) [inline message_definitions/v1.0/common.xml]: Not sure this is a valid tag. The % only holds true in one of three different use cases. In that case I think omitting the unit tag is a better approach.

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -4051,11 +4051,11 @@
...
       <field type="uint64_t" name="time_usec" units="us">Timestamp (micros since boot or Unix epoch)</field>
       <field type="uint16_t" name="flags" enum="ESTIMATOR_STATUS_FLAGS" display="bitmask">Integer bitmask indicating which EKF outputs are valid. See definition for ESTIMATOR_STATUS_FLAGS.</field>
-      <field type="float" name="vel_ratio">Velocity innovation test ratio</field>
-      <field type="float" name="pos_horiz_ratio">Horizontal position innovation test ratio</field>
-      <field type="float" name="pos_vert_ratio">Vertical position innovation test ratio</field>
-      <field type="float" name="mag_ratio">Magnetometer innovation test ratio</field>
-      <field type="float" name="hagl_ratio">Height above terrain innovation test ratio</field>
+      <field type="float" name="vel_ratio" units="Hz">Velocity innovation test ratio</field>
```
Comment (WickedShell) [inline message_definitions/v1.0/common.xml]: Estimator status message fields are Hz? The description of ratio and Hz are not things that I see how they go together.

File: message_definitions/v1.0/common.xml [hunk 3]
```diff
@@ -2881,16 +2881,16 @@
       <field type="uint32_t" name="h_acc" units="mm">Position uncertainty in meters * 1000 (positive for up).</field>
       <field type="uint32_t" name="v_acc" units="mm">Altitude uncertainty in meters * 1000 (positive for up).</field>
       <field type="uint32_t" name="vel_acc" units="mm">Speed uncertainty in meters * 1000 (positive for up).</field>
-      <field type="uint32_t" name="hdg_acc" units="mm">Heading / track uncertainty in degrees * 1e5.</field>
+      <field type="uint32_t" name="hdg_acc">Heading / track uncertainty in degrees * 1e5.</field>
```
Comment (WickedShell) [inline message_definitions/v1.0/common.xml]: I agree that mm is wrong, but why not add the correct unit?

Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml]: There is no such SI unit (1E5) https://www.bipm.org/en/publications/si-brochure/chapter3.html

File: message_definitions/v1.0/common.xml (hunk 1 above)
Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml]: Good point, I missed that. Will correct it.

File: message_definitions/v1.0/common.xml (hunk 2 above)
Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml]: The vel_ratio is already documented. I assume that the ratio information has to do with the update rate of some fields.

**Comment by amilcarlucas** [comment]
Thanks for reviewing @WickedShell . I addressed your issues now.

File: message_definitions/v1.0/common.xml (hunk 3 above)
Comment (WickedShell) [reply message_definitions/v1.0/common.xml]: There is also no such unit as degE7. All you need here is a degE5. Don't let the limitation of scaling stop you from having correct documentation.

**Comment by amilcarlucas** [comment]
This one needs to be merged first, so that the new CI tests introduced in mavlink/pymavlink#5 pass

**Comment by amilcarlucas** [comment]
The Travis tests only fail because it timedout while downloading python 3.5

**Comment by amilcarlucas** [comment]
All the new CI tests introduced by the latest pymavlink pass

**Comment by amilcarlucas** [comment]
@LorenzMeier The last master PR merge you did ( [HIGH_LATENCY2 message](https://github.com/mavlink/mavlink/pull/814) ) caused this PR to fail the CI tests.
I had to fix it, but one of the the reasons for this PR is to make the CI tests more inteligent, so that the other developers can fix the issues themselves.
Please merge this in before it gets more conflicts again.

**Comment by dagar** [comment]
I'm not familiar with the generate code, but this PR looks good to me.

**Comment by amilcarlucas** [comment]
I had the solve the conflicts again :(

**Comment by amilcarlucas** [comment]
Two CI tests pass, can someone restart the third ?

**Comment by dagar** [comment]
Restarted

**Comment by amilcarlucas** [comment]
@dagar Thanks, the CI tests pass again.

**Comment by amilcarlucas** [comment]
@dagar Thanks, for merging but it does require mavlink/pymavlink#7.

**Comment by amilcarlucas** [comment]
#843 is the permanent solution

**Pull request merged**